### PR TITLE
Fix makefile crds generation

### PR DESCRIPTION
**Pull request by PERES-Richard**
| Q               | A
| --------------- | ---
| Bug fix        | [x]
| License         | Apache 2.0

### What's in this PR?
Correct CRDs generation rule

### Why?
To deploy CRDs in both deploy directory and in Casskop and Multi Casskop Helm Crds directories.

### Checklist
- [x] Implementation tested
- [ ] Append changelog

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -185,7 +185,7 @@ docker-generate-k8s:
 		-v $(shell go env GOCACHE):/root/.cache/go-build:delegated --env GO111MODULE=on \
 		--env https_proxy=$(https_proxy) --env http_proxy=$(http_proxy) \
 		$(BUILD_IMAGE):$(OPERATOR_SDK_VERSION) /bin/bash -c 'operator-sdk generate k8s'
-	cp deploy/crds/* helm/*/crds/
+	cp deploy/crds/* */helm/*/crds/
```
Comment (cscetbon) [inline Makefile:188]: generate-k8s generates only zz files, so I think we can get rid of it here

Comment (PERES-Richard) [reply Makefile:188]: yeah I agree, done :)

File: Makefile [hunk 2]
```diff
@@ -194,6 +194,7 @@ docker-generate-crds:
 		--env https_proxy=$(https_proxy) --env http_proxy=$(http_proxy) \
 		$(BUILD_IMAGE):$(OPERATOR_SDK_VERSION) /bin/bash -c 'operator-sdk generate crds'
 	@sed -i '/\- protocol/d' deploy/crds/db.orange.com_cassandraclusters_crd.yaml
+	cp deploy/crds/* */helm/*/crds/
```
Comment (cscetbon) [inline Makefile:196]: here they are copied to multi-casskop but no longer to helm/ (at the root level) 👎

**Review by cscetbon — CHANGES_REQUESTED** [review]
see my comments

File: Makefile (hunk 2 above)
Comment (PERES-Richard) [reply Makefile]: Fixed by duplicating the `cp ` to hit both `*/hem` et `./helm/` dir

**Review by cscetbon — APPROVED** [review]

**Review by cscetbon — DISMISSED** [review]

**Review by cscetbon — APPROVED** [review]

**Pull request merged**